src/apply-templates/actionsCreator.ts

```diff
@@ -12,14 +12,18 @@ export class ActionsCreator {
         this.actionsHub.changeSelection.invoke(selectedTemplates);
     }
 
-    initialize(): void {
-        new TemplateStoreService().getTemplates().then(
-            templates => {
-                this.actionsHub.initialize.invoke(templates);
-            },
-            error => {
+    initialize(pullRequest: IActionPullRequest): void {
```

Comment:
I think it might be better to make these calls in parallel:

```ts
initialize(pullRequest: IActionPullRequest): void {
  Promise.all([
    new TemplateStoreService().getTemplates(),
    new ApplyTemplateService().getStatus(pullRequest)
  ].then([templates, prStatus] => {
    this.actionsHub.initializeTemplates.invoke(templates);
    this.actionsHub.initializeStatus.invoke(prStatus);
  }, error => {
    this.actionsHub.changeMode.invoke(Mode.Error);
  });
}
```